rtil/src/threads/ue/rebo/livesplit.rs

```diff
@@ -165,3 +172,103 @@ impl Run {
         Ok(())
     }
 }
+impl Layout {
+    pub fn new() -> LiveSplitLayout {
+        let mut layout = LiveSplitLayout::new();
+        layout.push(Component::Title(Title::new()));
+        layout.push(Component::Splits(Splits::new()));
+        layout.push(Component::DetailedTimer(Box::new(DetailedTimer::new())));
+        layout.push(Component::PreviousSegment(PreviousSegment::new()));
+        layout.push(Component::PossibleTimeSave(PossibleTimeSave::new()));
+        layout.push(Component::CurrentPace(CurrentPace::new()));
+        layout.push(Component::PbChance(PbChance::new()));
+        layout.push(Component::SumOfBest(SumOfBest::new()));
+        layout
+    }
+    fn settings() -> GeneralLayoutSettings {
+        LIVESPLIT_STATE.lock().unwrap().layout.general_settings().clone()
+    }
+    fn settings_mut() -> MappedMutexGuard<'static, GeneralSettings> {
+        MutexGuard::map(LIVESPLIT_STATE.lock().unwrap(), |x| x.layout.general_settings_mut())
+    }
```

Comment:
I don't think that we need `mapped_lock_guards` with all the functions being inside the same module. Currently, having the feature only allows writing `Layout::settings_mut().foo = bar` instead of `LIVESPLIT_STATE.lock().unwrap().foo = bar`. Depending on a nightly feature just for that isn't worth it IMO.